src/backends/pyplot.jl

```diff
@@ -499,32 +499,31 @@ function py_add_series(plt::Plot{PyPlotBackend}, series::Series)
                     :zorder => plt.n,
                     :cmap => py_linecolormap(series),
                     :linewidth => py_dpi_scale(plt, series[:linewidth]),
-                    :linestyle => py_linestyle(st, series[:linestyle])
+                    :linestyle => py_linestyle(st, series[:linestyle]),
+                    :norm => pycolors["Normalize"](; extrakw...)
                 )
-                if needs_colorbar
-                    kw[:norm] = pycolors["Normalize"](; extrakw...)
-                end
                 lz = collect(series[:line_z])
```

Comment:
Could you try to replace this line with
```julia
lz = _cycle(series[:line_z], 1:n)
```
?

Reply:
Nice, that did it

![multicolored](https://user-images.githubusercontent.com/7328215/31262287-d2e4fc04-aa0e-11e7-9ee5-412a1d98b385.png)